training.md

```diff
@@ -739,3 +739,26 @@ so our total batch size divided by the total number of chips cannot drop below
 $$\frac{4 \alpha^2}{F \cdot M_X \cdot M_Y},$$
 
 as we had derived above.
+
+### Appendix D - Unifying the "compute-bound" rules for various parallelism schemes
+
+To make it easier to remember, we can cast the various "compute-bound" rules from this section into a unified form which also highlights their physical origin. This form is:
```

Comment:
I don't quite know what "which also highlights their physical origin" means. "into a more intuitive, unified form"?

Reply:
Physical origin means essentially what I say in the final paragraph:

"Apart from being easier to remember, these matrices highlight the origin of these rules: for a given situation, we have a computation time $T_\text{math}$ and a transfer/communication time $T_\text{comms}$ or $T_\text{HBM}$. Both of these are functions of our matrix dimensions, and the relevant accelerator properties. Crucially, $T_\text{math}$ and $T_\text{comms}$/$T_\text{HBM}$ scale *differently* with respect to certain parameters, so by varying these parameters we can find a regime in which one or the other dominates."

The origin of these rules is the fact that Tmath an Tcomm scale differently wrt matrix parameters.  That's all I meant.

Reply:
Can we use a different word than physical then? What's physical about it?

Reply:
Could just kill the word physical.  "highlights their origin"?

Reply:
sg